src/sdl/sdl2graphics.cpp

```diff
@@ -115,7 +115,7 @@ namespace gcn
         SDL_SetTextureBlendMode(mTexture, SDL_BLENDMODE_BLEND);
     }
 
-    bool SDL2Graphics::pushClipArea(Rectangle area)
+    bool SDL2Graphics::pushClipArea(const Rectangle area)
```

Comment:
```suggestion
    bool SDL2Graphics::pushClipArea(const Rectangle& area)
```
With declaration too ;-)